### ISSUE TITLE
[Question] Widget AttachmentsTableWithPreview

### ISSUE THREAD

**Issue by Arekusei** [Nemmo/yii2-attachments#41]
What is it the hell code?
```php
        $hasFileBehavior = false;
        foreach ($this->model->getBehaviors() as $behavior) {
            if ($behavior->className() == FileBehavior::className()) {
                $hasFileBehavior = true;
            }
        }
```
Why not used `instanceof`?

**Comment by Nemmo** [comment]
Because it works! xD But `instanceof` is really the right way.

**Comment by Arekusei** [comment]
It works, but not extendable.

**Comment by Arekusei** [comment]
And for this code it will be good break foreach after `$hasFileBehavior = true`.
```php
        $hasFileBehavior = false;
        foreach ($this->model->getBehaviors() as $behavior) {
            if ($behavior instanceof FileBehavior) {
                $hasFileBehavior = true;
                break;
            }
        }

**Comment by Nemmo** [comment]
Fixed 3ed19c5e2e5b424c7bd0c5dfbff17e49a52f57a7

**Comment by Arekusei** [comment]
Thanks
